### ISSUE TITLE
missing configurable option csiPluginName

### ISSUE THREAD

**Issue by wesselvdv** [eddycharly/terraform-provider-kops#213]
I am missing the option to configure the following property:

```yaml
  cloudConfig:
    openstack:
      blockStorage:
        csiPluginImage: docker.io/k8scloudprovider/cinder-csi-plugin:v1.20.3
```

Without this option, I can't deploy kops.

**Comment by eddycharly** [comment]
Isn't it a 1.20 option only ?
I started working on 1.20 support, the current version is based on kOps 1.19.3 (and i feel like `csiPluginImage` is not present in this version).

I will try to publish an alpha version for 1.20 tonight.

**Comment by eddycharly** [comment]
It is present in 1.20 indeed (see https://github.com/eddycharly/terraform-provider-kops/blob/kops-1-20/docs/resources/cluster.md#openstack_block_storage_config) the corresponding name is not correct though `cs_iplugin_image`.

**Comment by wesselvdv** [comment]
Awesome! I will definitely try that 1.20 one.

**Comment by eddycharly** [comment]
I published version `1.20.0-alpha.1`. Beware that i didn't test it at all yet, not even on AWS.
I fixed the `csi_plugin_image` attribute name, let me know how things go.

**Comment by eddycharly** [comment]
Closing, it's now supported.
